### DISCUSSION TITLE
Unable to download slides

### DISCUSSION THREAD

**Discussion by teddcp2** [mrdbourke/pytorch-deep-learning#209] [General]
Hi all,

I am unable to download any slides from the repo and it seems to be a DNS issue. I tried to change it to 8.8.8.8, but still it's not working.
Can someone please share any alternative link (Gdrive) to the slides ?

Any help would be much appreciated.

![github error](https://user-images.githubusercontent.com/68907349/206835883-d5064890-f3e2-45cf-8376-d34965638d14.PNG)

**Comment by mrdbourke** [comment]
Hi @teddcp2,

Sometimes GitHub lags out and can't offer the slides to download.

You can find all of the slides here on Dropbox: https://www.dropbox.com/sh/crcw2s5vsca7h7f/AACiI9HQ1Epu-Dq1bOGqVxzAa?dl=0 

Let me know if it works :)
